rcgen/src/key_pair.rs

```diff
@@ -377,6 +421,16 @@ impl TryFrom<Vec<u8>> for KeyPair {
 	}
 }
 
+/// The key size used for RSA key generation
+#[cfg(all(feature = "crypto", feature = "aws_lc_rs", not(feature = "ring")))]
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+#[non_exhaustive]
+pub enum RsaKeySize {
+	_2048,
```

Comment:
Nit: the `_` prefix looks unidiomatic to me, but not sure I have a better idea.

Reply:
Yeah, not really perfect. `r#2048` doesn't work, but I like `_2048` more than `Rsa2048`.